### PR TITLE
@xtina-starr => Support partner featured bio on artist biography page

### PR DESCRIPTION
**Pull request by mzikherman**
This updates the biography page for an artist to support featured bios, through Metaphysics. MG version of https://github.com/artsy/force/pull/314.

Pretty straightforward, safe to merge. Next up, trying to figure out how to 'know' whether to render the bio tab on the main artist page, in the event of no Artsy bio but an existing featured partner bio. Unless the whole page is rewritten to use Metaphysics...I may have to add that call :(

### CODE REVIEW

**Comment by mzikherman** [comment]
Closes part of https://github.com/artsy/microgravity/issues/39

**Comment by mzikherman** [comment]
Ok, pr'ed a hacky, but necessary commit that will fetch/set the featured partner bio if there isn't an Artsy one. This way, the template can render the bio link if present.

Since existing code relies on Metaphysics to orchestrate (and it still needs to make the partner artist call), I don't see a way around this.

It's a pretty light/cached call though, so no biggie 👍

File: apps/artist/routes.coffee [hunk 1]
```diff
@@ -1,23 +1,39 @@
...
-      res.locals.sd.ARTIST = artist.toJSON()
-      showAuctionLink = artist.get('display_auction_link')
-      res.render 'page', artist: artist, sort: req.query?.sort, showAuctionLink: showAuctionLink
+      artist.maybeFetchAndSetFeaturedBio =>
+        res.locals.sd.ARTIST = artist.toJSON()
+        showAuctionLink = artist.get('display_auction_link')
+        res.render 'page', artist: artist, sort: req.query?.sort, showAuctionLink: showAuctionLink
     error: res.backboneError
 
 module.exports.biography = (req, res, next) ->
```
Comment (mzikherman) [inline apps/artist/routes.coffee:31]: stand-alone bio page uses metaphysics

File: apps/artist/templates/header.jade [hunk 2]
```diff
@@ -1,7 +1,11 @@
+-var hometown = artist.hometown || artist.get('hometown')
```
Comment (mzikherman) [inline apps/artist/templates/header.jade:1]: to account for this being used in a metaphysics/plain object, or backbone model context

File: models/artist.coffee [hunk 3]
```diff
@@ -23,6 +23,16 @@ module.exports = class Artist extends Backbone.Model
   hasImage: (size = 'tall') ->
     size in (@get('image_versions') || [])
 
+  maybeFetchAndSetFeaturedBio: (cb = ->) ->
```
Comment (mzikherman) [inline models/artist.coffee:26]: Tries to set the blurb to the featured partner bio one, if there is one (and if there isn't an Artsy blurb).

**Comment by mzikherman** [comment]
Closes all of https://github.com/artsy/microgravity/issues/39

File: apps/artist/templates/header.jade (hunk 2 above)
Comment (xtina-starr) [reply apps/artist/templates/header.jade:1]: Nice I was going to ask about this.

**Comment by xtina-starr** [comment]
⭐️

**Pull request merged**